codegen-serde/build.gradle.kts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+
+plugins {
+    kotlin("jvm")
+    `maven-publish`
+}
+
+description = "Plugin to generate `serde` implementations. NOTE: This is separate from generalized serialization."
+extra["displayName"] = "Smithy :: Rust :: Codegen Serde"
+extra["moduleName"] = "software.amazon.smithy.rust.codegen.client"
+
+group = "software.amazon.smithy.rust.codegen.serde"
+version = "0.1.0"
+
+val smithyVersion: String by project
+
+dependencies {
+    implementation(project(":codegen-core"))
+    implementation(project(":codegen-client"))
+    implementation(project(":codegen-server"))
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
Aren't we on v17 now?

Reply:
oh dono, I copied this from codegen core, can change it to whatever.